package.json

```diff
@@ -101,51 +101,53 @@
     "eslint-plugin-react": "6.9.0",
     "esprima": "3.1.3",
     "extract-text-webpack-plugin": "1.0.1",
-    "file-loader": "0.8.5",
+    "file-loader": "1.1.6",
```

Comment:
How did you arrive at using these versions? Is it the latest, or just a version that works with this new version of node?

Reply:
AFAIK all of them are the latest version at the time we updated 👍

Reply:
I took everything that was related to webpack, upgraded it to the latest version and wrote a webpack config from scratch